<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1241</width>
    <height>810</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolBar
{
	spacing:5px;
	border: none;
}

QToolButton
{
	margin: 2px 2px 2px 2px;
}

QLabel#itemNameLabel
{
	font: 16px;
}

QLabel#itemCodeLabel
{
	color: #999999;
}

QLabel#currentValueLabel
{
	font: bold 16px;
	color: #990000;
}

QGroupBox
{
	border: none;
}</string>
  </property>
  <property name="iconSize">
   <size>
    <width>16</width>
    <height>16</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="leftWidget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>274</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="fundListLabel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">padding: 2px;font-size: 14px;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Sunken</enum>
       </property>
       <property name="text">
        <string>所选基金列表</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="fundListWidget">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="rightWidget" native="true">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>631</width>
      <height>471</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="fundTabelLabel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">padding: 2px 10px;
font-size: 14px;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Sunken</enum>
       </property>
       <property name="text">
        <string>asdfsadf</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="fundTableWidget">
       <property name="editTriggers">
        <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
       </property>
       <property name="gridStyle">
        <enum>Qt::PenStyle::DotLine</enum>
       </property>
       <property name="rowCount">
        <number>3</number>
       </property>
       <property name="columnCount">
        <number>3</number>
       </property>
       <row/>
       <row/>
       <row/>
       <column/>
       <column/>
       <column/>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1241</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件(F)</string>
    </property>
    <addaction name="actionLogin"/>
    <addaction name="actionLogout"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助(H)</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>工具(&amp;T)</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionSync"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionSetup"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>操作(O)</string>
    </property>
    <addaction name="actionTradeFund"/>
    <addaction name="separator"/>
    <addaction name="actionAnalyzeIndex"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu"/>
   <addaction name="menuTool"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLogin"/>
   <addaction name="actionLogout"/>
   <addaction name="actionTradeFund"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionLogin">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ContactNew"/>
   </property>
   <property name="text">
    <string>登录(&amp;L)...</string>
   </property>
   <property name="iconText">
    <string>登录</string>
   </property>
   <property name="toolTip">
    <string>登录应用</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::SystemShutdown"/>
   </property>
   <property name="text">
    <string>退出(&amp;Q)</string>
   </property>
   <property name="iconText">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出当前应用程序</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::HelpAbout"/>
   </property>
   <property name="text">
    <string>关于(&amp;A)...</string>
   </property>
   <property name="toolTip">
    <string>关于...</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::UserAvailable"/>
   </property>
   <property name="text">
    <string>注销(&amp;O)</string>
   </property>
   <property name="iconText">
    <string>注销</string>
   </property>
   <property name="toolTip">
    <string>注销账户</string>
   </property>
  </action>
  <action name="actionSync">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::MediaPlaylistRepeat"/>
   </property>
   <property name="text">
    <string>数据更新(&amp;R)...</string>
   </property>
   <property name="iconText">
    <string>数据更新</string>
   </property>
   <property name="toolTip">
    <string>数据更新...</string>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
  </action>
  <action name="actionSetup">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentProperties"/>
   </property>
   <property name="text">
    <string>设置(&amp;S)</string>
   </property>
   <property name="iconText">
    <string>设置</string>
   </property>
   <property name="toolTip">
    <string>设置</string>
   </property>
  </action>
  <action name="actionAnalyzeIndex">
   <property name="text">
    <string>指数分析(&amp;I)...</string>
   </property>
   <property name="iconText">
    <string>指数分析</string>
   </property>
   <property name="toolTip">
    <string>指数分析(I)...</string>
   </property>
  </action>
  <action name="actionTradeFund">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::MediaPlaylistShuffle"/>
   </property>
   <property name="text">
    <string>基金交易(O)...</string>
   </property>
   <property name="iconText">
    <string>基金交易</string>
   </property>
   <property name="toolTip">
    <string>基金交易</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionLogin</sender>
   <signal>triggered()</signal>
   <receiver>mainWindow</receiver>
   <slot>onActionLoginTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>630</x>
     <y>404</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>mainWindow</receiver>
   <slot>onActionExitTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>630</x>
     <y>404</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLogout</sender>
   <signal>triggered()</signal>
   <receiver>mainWindow</receiver>
   <slot>onActionLogoutTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>630</x>
     <y>404</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSync</sender>
   <signal>triggered()</signal>
   <receiver>mainWindow</receiver>
   <slot>onActionSetupDataTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>524</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSetup</sender>
   <signal>triggered()</signal>
   <receiver>mainWindow</receiver>
   <slot>onActionSetupFundTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>524</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAnalyzeIndex</sender>
   <signal>triggered()</signal>
   <receiver>mainWindow</receiver>
   <slot>onActionAnalyzeIndexTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>721</x>
     <y>512</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionTradeFund</sender>
   <signal>triggered()</signal>
   <receiver>mainWindow</receiver>
   <slot>onActionTradeFundTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>721</x>
     <y>512</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>mainWindow</receiver>
   <slot>onActionAboutTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>721</x>
     <y>512</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fundListWidget</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>mainWindow</receiver>
   <slot>onFundListWidgetItemSelectionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>356</y>
    </hint>
    <hint type="destinationlabel">
     <x>450</x>
     <y>1003</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onActionLoginTriggered()</slot>
  <slot>onActionLogoutTriggered()</slot>
  <slot>onActionExitTriggered()</slot>
  <slot>onActionSetupDataTriggered()</slot>
  <slot>onActionSetupFundTriggered()</slot>
  <slot>onActionAnalyzeIndexTriggered()</slot>
  <slot>onActionTradeFundTriggered()</slot>
  <slot>onActionAboutTriggered()</slot>
  <slot>onFundListWidgetItemSelectionChanged()</slot>
 </slots>
</ui>
